runtime/integration-tests/src/runtime.rs

```diff
@@ -273,10 +273,10 @@ mod mandala_only_tests {
 
 			assert_eq!(base_fee, 1_000_000_000);
 			assert_eq!(len_fee, 500_000_000);
-			assert_eq!(adjusted_weight_fee, 4_592_000);
+			assert_eq!(adjusted_weight_fee, 0);
```

Comment:
not sure why fee is changed? @zjb0807

Reply:
I found the reason is remark call weight changed to:
https://github.com/paritytech/substrate/blob/919e304dd6f3c11a55ee125498e8e531ce343805/frame/system/src/weights.rs#L100-L102

while previous is 574_000
https://github.com/paritytech/substrate/blob/19162e43be45817b44c7d48e50d03f074f60fbf4/frame/system/src/weights.rs#L99-L103